src/components/tabs/tabs.module.css

```diff
@@ -23,20 +26,28 @@ keeps us from having to set styles in the JSX.
 ref: https://twitter.com/heydonworks/status/408263890284191745
 ref: https://twitter.com/MarcoInEnglish/status/363284092806119424
 */
-.tabPanel[role='tabpanel'] {
+.tabPanel {
```

Comment:
I've removed the `role` selector here to allow styles for this selector to be grouped (as with the additional selector part, `composes` cannot be used).

Reply:
🚧 I think since we need the `[role='tabpanel']` selector to stay, we should just move the `composes: g-focus-ring-from-box-shadow from global` to the JSX as as a hardcoded string class name on the element.

Reply:
Yep that's one way to put it! The purpose is to tie together the state of an element with its style using selectors readily available to us where possible. This practice encourages us to determine the correct roles, states, and properties for elements to be accessible and then it helps us maintain those roles, states, and properties in the longer term. I would consider this practice to be required (with an understanding that it takes time to get used to).